### DISCUSSION TITLE
apps to add, please.

### DISCUSSION THREAD

**Discussion by fabcard** [ChrisTitusTech/winutil#137] [Ideas]
Hello,
First, thank you for the great tool.
I have a few apps to add to the installation section, please:

K-LiteCodecPack Standard
Sandboxie Plus
ProcessLasso
InternetDownloadManager
WinMerge
Microsoft Visual C++ 2015-2022 Redistributable (x64)
Microsoft Visual C++ 2015-2022 Redistributable (x86)
Microsoft Windows Desktop Runtime (x64)
Microsoft Windows Desktop Runtime (x86)

Again, thanks in advance.

**Comment by HCCIT** [comment]
Please add:

clink
starship
pandoc
virt-viewer
spice-agent
processexplorer
okular
powertoys
autoruns
peazip
everything
everything-toolbar
OnlyOffice
msedgeredirect
Marktext
Wireguard
pdfarranger
audacious
nuclear
zettlr
filezilla
drawio
joplin
strawberry
kde connect
nextcloud client

**Comment by ChrisTitusTech** [comment]
Good lists, I'll be sure and put these in the queue.

**Reply by ghost** [reply]
I was wondering if you could add some of the following apps:


TronScript
Process Hacker
VirtualBox
VMWare
QeMu
Epic Privacy Browser
ChatGPT Desktop (https://github.com/lencx/ChatGPT/releases)
Resource Hacker
SuperTux
SuperTuxKart
Riot Games Launcher
Battle Net
Godot
Unreal Engine
Rust
8GadgetPack
Minecraft
Opera
OperaGX

**Comment by cdrivex4** [comment]
forgive my relative lack of intelligence on the subject...but..if we have winget and choco as two sources...it seems to me it might be possible to use this script to build a list of apps on a given pc..as a separate table.. and if they exist in either/or location/repository a mechanism to propagate updates and perhaps install app not on the "..titus" list exists..this could really be useful if there was a scroll-able list say for example at the very far right of the existing table and i get it...your not going to get every esoteric program out there..thats for the devs to add themselves to winget repo or choco...you see this stems from me trying to get chatGPt to give me a script to backup/deploy apps on multiple systems using winget and failing...this mechanism is a lot more robust in parsing installed software ....you could still have the "titus list" but also at the same time you would expand the scope to be able to pull software not validated..see looking at that list I see procexp and that is part of sysinternals..theres more to systinternal than just procexp..would you just then pull procexp or sysinternal through winget...then you have nested applications on a list which currently doesnt really work for it..please note love tool.i can imagine a time where perhaps im asked to help a friend I remote in, run titus tool and boom..admin stuff done.. thanks all for the progress so far....ok hold up...upgrade all, it updating all apps and not respecting what has been checked on the list..

**Comment by xaksin** [comment]
Apologies: This is my first ever post in github.  Feedback and guidance welcome.

RE: Winget section
The "UPGRADE ALL" in the winget section has an annoying problem.
Sometimes it is important NOT to upgrade certain programs for a variety of reasons on a machine.
**How can you tell it not to upgrade specific programs or to allow multiple versions in certain instances?**

Some examples:
Hardware compatibility may require java 32-bit and when upgraded to 64-bit the hardware stops working.
Old hardware limitations may not allow a program to work past a certain version.  e.g. ShotCut
